appcelerator/titanium_mobile
# [TIMOB-7347] Using getters for ContentOffset instead of dynaprops to avoid error message from Kroll Layer when trying to remember

**Testing Instructions:**

Run KitchenSink for the following tests
     Base UI > Views > Scroll Views > Scrolling Zoom+Pinch
     Base UI > Views > Scroll Views > Scrolling Drag Start&E
     Base UI> Views> Scroll Views>Basic:
     Base UI> Views> Scroll Views>Scrolling tabs:

Additional test for sanity:

Test the following [app.js](https://gist.github.com/1713839) file :
     Check log to see if you can see type of content Offset to be TiPoint and the values being returned properly.
